ditto/writers/opendss/write.py

```diff
@@ -135,127 +143,110 @@ def float_to_str(self, f):
         d1 = ctx.create_decimal(repr(f))
         return format(d1, "f")
 
-    def write(self, model, **kwargs):
+    def write(self, model, separate_feeders = False, separate_substations = False, write_taps=False, verbose=False):
         """General writing function responsible for calling the sub-functions.
 
+        Note: re.sub('[^0-9a-zA-Z]+', '_', object_name) is used to fix node/bus names for OpenDSS, 
+            which uses dots for phase designation and spaces for paramater delimiters.
+
         :param model: DiTTo model
         :type model: DiTTo model
-        :param verbose: Set verbose mode. Optional. Default=False
-        :type verbose: bool
+        :param separate_feeders: Whether or not to create subfolders for different feeders. Optional. Default=False
+        :type separate_feeders: bool
+        :param separate_substations: Whether or not to create subfolders for different substations. Optional. Default=False
+        :type separate_substations: bool
```

Comment:
@tarekelgindy can you please document what the folder structure and Master.dss redirects look like when `separate_feeders` and/or `seperate_substations` are `True`?